src/nupic/types/Types.h

```diff
@@ -36,9 +36,10 @@
 
 #if defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC) && defined(NDEBUG)
 #pragma warning( disable : 4244 ) // conversion from 'double' to 'nta::Real', possible loss of data (LOTS of various type combinations)
-#pragma warning( disable : 4305 ) // truncation from 'double' to 'nta::Real', possible loss of data (LOTS of various type combinations)
 #pragma warning( disable : 4251 ) // needs to have dll-interface to be used by clients of class 
 #pragma warning( disable : 4275 ) // non dll-interface struct used as base for dll-interface class
+#pragma warning( disable : 4305 ) // truncation from 'double' to 'nta::Real', possible loss of data (LOTS of various type combinations)
+#pragma warning( once : 4838 ) // narrowing conversions
```

Comment:
how is 4305 different from 4244 above? And I don't understand 4838 well enough - where does it occur in the code?